examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3088,6 +3088,7 @@ var GLTFLoader = ( function () {
 
 			if ( nodeDef.name !== undefined ) {
 
+				node.originalName =  nodeDef.name ;
 				node.name = PropertyBinding.sanitizeNodeName( nodeDef.name );
```

Comment:
I'd rather not attach new properties to native three.js objects. Perhaps `node.userData.name = nodeDef.name`?

/cc @takahirox

Reply:
Yeah, if we store an original name, I prefer `.userData`.

Reply:
Yep, you're absolutely right.